R/render.R

```diff
@@ -367,6 +367,15 @@ render <- function(input,
              "of this document to HTML.", call. = FALSE)
       }
     }
+    # if this isn't latex and there are latex dependencies then flag an error
+    if (!(identical(output_format$pandoc$to, "pdf") ||
```

Comment:
`to` cannot be `pdf`: it can only be `latex` or `beamer` (there may be other LaTeX formats in the future in Pandoc)

I'd say let's get rid of the check here for now. It makes sense but it can be messy to do it completely right, e.g. the file extension could be `.tex`, which is legitimate to have LaTeX dependencies.